docs/installation/_index.md

```diff
@@ -82,9 +72,28 @@ Proceed to [install `viam-micro-server`](/installation/#install-viam-micro-serve
 {{% /tab %}}
 {{< /tabs >}}
 
+### Operating System installation
+
+`viam-server` can run on **any computer that runs a [supported operating system](#platform-requirements)**, including single-board computers (SBCs) that provide access to GPIO pins and can be configured as [board](/components/board/) components.
```

Comment:
```suggestion
`viam-server` can run on **any computer that runs a [supported operating system](#platform-requirements)**, including single-board computers (SBCs) running 64-bit Linux.
```
I feel like this sentence further adds to the confusion of https://viam.atlassian.net/browse/DOCS-2892